resources/options.html

```diff
@@ -100,30 +100,29 @@
         </div>
     </div>
 
-    <!--TODO: Make text understandable to the user (will be done when backend is properly working.-->
     <div class="col s12 m6">
-        <div class="card" id="donotwatch">
+        <div class="card" id="doNotWatch">
             <div class="card-content">
-                <span class="card-title">Feature selection</span> <br>
+                <span class="card-title">Tracking selection</span> <br>
                 <p style="font-family: Roboto,sans-serif; font-size:11pt">
-                    Certain tracking features can be disabled. Make a selection below.
+                    The checked elements and events will be tracked while Octopeer is running.
                 </p>
-                <ul id="donotwatch_sub">
+                <ul id="doNotWatch_sub">
                     <li>
-                        <input type="checkbox" id="dnwscreen">
-                        <label for="dnwscreen">Elements on the screen. </label>
+                        <input type="checkbox" id="dnwScreen" checked>
+                        <label for="dnwScreen">Elements on the screen. </label>
                     </li>
                     <li>
-                        <input type="checkbox" id="dnwhover">
-                        <label for="dnwhover">Elements I hover over with the cursor. </label>
+                        <input type="checkbox" id="dnwHover" checked>
+                        <label for="dnwHover">Elements I hover over with the cursor. </label>
```

Comment:
Silly thing... Shouldn't this be "mouse"? Cause the cursor is an indicator where you type XD